usr/share/rear/format/USB/default/30_format_usb_disk.sh

```diff
@@ -9,7 +9,26 @@ if [[ "$answer" == "Yes" || "$FORCE" ]]; then
 
     if [[ "$EFI" == "y" ]]; then
         LogPrint "The --efi toggle was used with format - make an EFI bootable USB disk"
-        echo "Yes" | parted -s $RAW_USB_DEVICE -- mklabel gpt mkpart primary 0 100Mib mkpart primary 100Mib 100% >&2
+        
+        # Prompt user for size of EFI partition on USB disk
+        # Pressing Enter (\n) will use default value from default.conf
+        echo -n "Please enter size of EFI partition on USB device in MB [default ${USB_UEFI_PART_SIZE} MB]: "
+        read efi_part_size
+        
+        # Check if user entered unsigned integer larger than 0
+        if [[ "${efi_part_size}" =~ ^[0-9]+$ && ${efi_part_size} > 0 ]]; then
```

Comment:
Primary intention here was only to disallow enter zero as an value,  as I'm not sure how parted would handle zero as input value and was worried that such corner case could cause problems.
As user I'd welcome if I could choose arbitrary value for EFI partition size. Making this value dependent on default value sounds confusing for me.